screens/quiz-categories/category.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/stylesheets/components.css">
+    <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/landing.css">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="./category.css">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" />
+    <title>Categories</title>
+
+</head>
+
+<body>
+    <!-- Navbar -->
+    <nav class="nav">
+        <div class="nav-logo">
+            <span class="nav-header txt-theme-8">QuizMaster</span>
+        </div>
+        <span id="btn-nav" class="nav-btn"><i class="fas fa-chevron-circle-down fa-lg"></i></span>
+        <ul id="list-nav" class="nav-list nav-list-hide">
+            <li class="list-item txt-bold">Home</li>
+            <li class="list-item  list-item-active txt-bold">Categories</li>
+            <li class="list-item  txt-bold">logout</li>
+        </ul>
+    </nav>
+    <h2 class="txt-center txt-theme-3 top-heading">Choose a Category</h2>
+
+    <!-- Movies Categories section -->
+    <p class="category-heading txt-big l-sp-2">MOVIES&nbsp;&nbsp;<i class="fas fa-arrow-right"></i></p>
+    <section class="category-component flex-box">
+        <div class="card">
+            <img src="/images/bolly2.jpg" class="" alt="">
```

Comment:
just a suggestion to give alt attribute some description

Reply:
Thanks for pointing it out